src/plugins/visualizations/public/expressions/visualization_function.ts

```diff
@@ -107,6 +116,7 @@ export const visualization = (): ExpressionFunctionVisualization => ({
         inspectorAdapters,
         queryFilter: getFilterManager(),
         forceFetch: true,
+        aggConfigs,
```

Comment:
Could we call this `aggs` here instead of `aggConfigs`, as this is consistent with the request handler inside of `esaggs.ts`, and also with our previous implementation?

Reply:
OK, I'll do that.